rudder-web/src/main/scala/com/normation/rudder/web/components/RuleGrid.scala

```diff
@@ -314,12 +314,18 @@ class RuleGrid(
             }
             }
           } catch {
-            case e:Exception => Alert("Error while trying to move group :"+e)
+            case e:Exception =>
+              val msg = s"Error while trying to apply directive ${directiveApp.directive.id.value} on visible Rules"
+              logger.error(s"$msg, cause is: ${e.getMessage()}")
+              logger.debug(s"Error details:")
+              e.getStackTrace().foreach(st => logger.debug(st))
```

Comment:
You should juste use: logger.debug(s"Error details:", e) - it will duplicates the exception first line, but it's ok since the log will be in debug and so, when debugging it is easier to understand the whole exception.